bridges/snowbridge/parachain/pallets/inbound-queue/Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "snowbridge-inbound-queue"
 description = "Snowbridge Inbound Queue"
-version = "0.1.1"
-edition = "2021"
-authors = ["Snowfork <contact@snowfork.com>"]
-repository = "https://github.com/Snowfork/snowbridge"
+version = "0.9.0"
+edition.workspace = true
+authors.workspace = true
```

Comment:
I realize now that setting `authors.workspace = true` is going mark Parity as the authors of this crate (similarly for other crates), which is not really ideal.

Reply:
I don't see it necessary that the Snowfork pallets be released/published to crates.io from the `polkadot-sdk` repo.

You could directly publish them from your repo.

Then, polkadot-sdk test runtimes can even consume them from crates.io directly instead of them being copied over to this repo.

Reply:
So you are saying perhaps we should remove our subtree in the `bridges` dir and just import from crates.io?

Reply:
The main challenge is that your pallets depend on some `polkadot-sdk` primitives crates, then `polkadot-sdk` runtimes depend on your crates but also on same primitive crates. This (most times) leads to pains with Cargo.lock issues coming from "incompatible" versions of same (primitives) crate being used.

Looks like a non-trivial problem, so I don't think we should block on it now, but options might be worth exploring in the future.